SPECS/redis/redis.spec

```diff
@@ -75,14 +75,24 @@ exit 0
 %files
 %defattr(-,root,root)
 %license COPYING
-%dir %attr(0750, redis, redis) /var/lib/redis
-%dir %attr(0750, redis, redis) /var/opt/%{name}/log
+%dir %attr(0750, redis, redis) %{_sharedstatedir}/redis
+%dir %attr(0750, redis, redis) %{_var}/opt/%{name}/log
 %attr(0750, redis, redis) %{_var}/log/%{name}
 %{_bindir}/*
 %{_libdir}/systemd/*
 %config(noreplace) %attr(0640, %{name}, %{name}) %{_sysconfdir}/redis.conf
 
 %changelog
+* Wed Jun 15 2022 Muhammad Falak <mwani@microsoft.com> - 6.2.7-1
+- Bump version to 6.2.7 to address CVE-2022-24736
+
+* Mon Feb 07 2022 Muhammad Falak <mwani@microsoft.com> - 6.2.6-1
```

Comment:
Maybe exclude the intermediates here and just add your name and the bump to 6.2.7-1